config/protocol.mainnet.yml

```diff
@@ -1,5 +1,5 @@
 ProtocolConfiguration:
-  Magic: 5195086
+  Magic: 860833102
```

Comment:
`MainNet` constant from `neo-go/pkg/config/netmode/netmode.go` should also be updated.

Reply:
I _always_ keep forgetting it, meh.

Reply:
Fixed.